samples/valueentity-counter/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "valueentity-counter",
+  "version": "0.0.1",
+  "type": "module",
+  "engines": {
+    "node": ">=13.0.0",
+    "npm": ">=6.0.0"
+  },
+  "dependencies": {
+    "@grpc/grpc-js": "^1.3.0",
+    "@grpc/proto-loader": "^0.1.0",
+    "@lightbend/akkaserverless-javascript-sdk": "0.0.0",
+    "google-protobuf": "^3.0.0",
+    "protobufjs": "^6.11.2"
+  },
+  "devDependencies": {
+    "chai": "^4.3.3",
+    "jsdoc": "^3.6.3",
+    "mocha": "^8.3.1",
+    "uglify-js": "^3.7.7"
+  },
+  "config": {
+    "dockerImage": "my-docker-repo/valueentity-counter",
+    "sourceDir": "./src",
+    "testSourceDir": "./test",
+    "protoSourceDir": "./proto",
+    "generatedSourceDir": "./lib/generated"
+  },
+  "scripts": {
+    "start": "node src/index.js",
+    "test": "mocha ./test",
+    "integration-test": "mocha ./integration-test",
+    "build": "compile-descriptor proto/counter_api.proto proto/counter_domain.proto"
```

Comment:
The package.json generated from template has some extra docker commands. Did you remove it intentionally?

Reply:
yes, I have had to remove `akkasls-scripts` and the docker commands are dependent on it.